src/interpreter.rs

```diff
@@ -21,6 +21,15 @@ use memory::{Memory, Pointer};
 use primval::{self, PrimVal};
 
 const TRACE_EXECUTION: bool = true;
+static mut INDENTATION: usize = 0;
```

Comment:
You could use `static INDENTATION: AtomicUsize = ATOMIC_USIZE_INIT;` to avoid unsafety.